### PR TITLE
Translation: Add more translators to translator list

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Adds translators from Weblate to the util.h list. For now, they are added manually, I think in future we should only list main contributors there and others in the ChangeLog only.

CHANGELOG: SKIP

**Context: Fixes an issue?**

No. But part of the release process.

**Does this change need documentation? What needs to be documented and how?**

No. Maybe some discussion.

**Status of this Pull Request**

Needs  review.

**What is missing until this pull request can be merged?**

Review

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [ ] I've filled all the content above

### CODE REVIEW

File: src/util.cpp [hunk 1]
```diff
@@ -726,17 +728,23 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                               tr ( "Swedish" ) +
                               "</b></p>"
                               "<p>Daniel (<a href=\"https://github.com/genesisproject2020\">genesisproject2020</a>)</p>"
+                              "<p>tygyh (<a href=\"https://github.com/tygyh\">tygyh</a>)</p>"
+                              "<p>Allan Nordhøy (<a href=\"https://hosted.weblate.org/user/kingu/\">kingu</a>)</p>"
                               "<p><b>" +
                               tr ( "Korean" ) +
                               "</b></p>"
                               "<p>Jung-Kyu Park (<a href=\"https://github.com/bagjunggyu\">bagjunggyu</a>)</p>"
+                              "<p>이정희 (<a href=\"https://hosted.weblate.org/user/MarongHappy/\">MarongHappy</a>)</p>"
```
Comment (ann0see) [inline src/util.cpp:737]: This doesn't show up for me correctly as I don't have support for Korean installed on my machine.

Comment (ann0see) [reply src/util.cpp:737]: Seems ok after I found the correct package.

**Review by pljones — APPROVED** [review]

**Comment by pljones** [comment]
```
$ ./jamulus-upstream/tools/get_release_contributors.py --from release --to next-release --repo jamuluswebsite-upstream
WARNING bd6def65b664582db7a06e85eafc3643677a7a08 has not github author, saving as empty
Website contributors/translators: @ann0see @gilgongo @ignotus666 @jujudusud @mcfnord @pljones @trebmuh @weblate
```
We'll get there...

**Review by hoffie — APPROVED** [review]

**Comment by ann0see** [comment]
> For now, I think we should manually add to util.h -- until the automation can do it. I also think that all contributors should have equal status and be in the about box.

Some aren't added as I didn't find any profile info for them. But they are in the changelog.

**Pull request merged**